.github/workflows/tests.yml

```diff
@@ -46,3 +46,11 @@ jobs:
     - uses: actions/checkout@v2
     - name: Check the README.md file is up-to-date
       run: sh scripts/check-readme.sh
+
+  wasm_build:
+    name: wasm-build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --example web_app
```

Comment:
I don’t think `cargo build —example web_app` will catch the error that happened in #135. When I ran this against the current code this build completed without error. The error didn’t show up until building with `cargo build --lib --release --target wasm32-unknown-unknown`

Reply:
That's right. The command that should be used is `cargo build --example web_app --target wasm32-unknown-unknown`

Reply:
```suggestion
      run: cargo check --example web_app --target wasm32-unknown-unknown
```

Reply:
My bad, thanks to both of you

Reply:
I have an error with this command but not the expected one

Reply:
https://github.com/meilisearch/meilisearch-rust/runs/2655215413?check_suite_focus=true

Reply:
> I have an error with this command but not the expected one

I'm not sure if this will work in GitHub actions, but try

```yaml
steps:
- uses: actions/checkout@v2
- name: Build
  run:
    rustup target add wasm32-unknown-unknown`
    cargo build --example web_app
```

Reply:
Hahaha yes! Sorry hard day for me today 😂

Reply:
Thaaaaaanks!!! Hard day for me 😂

Reply:
It faiiiils! So it works!!! 🎉